src/MemberReportExtension.php

```diff
@@ -23,6 +38,19 @@ class MemberReportExtension extends DataExtension
         'PermissionsDescription' => 'Text'
     );
 
+    /**
+     * Retrieves the most recent successful LoginAttempt
+     */
+    public function getLastLoggedIn()
+    {
+        $lastTime = $this->owner->LoginAttempts()
+            ->filter('Status', 'Success')
+            ->sort('Created', 'DESC')
+            ->first();
+
+        return $lastTime ?: _t(__CLASS__ . '.NEVER', 'never');
```

Comment:
I kind of feel like this should be `Never` - also do we need to run the text collector?